src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2841,6 +2841,209 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-jparse"
+    title={<InlineCode>jparse(attribute [, path])</InlineCode>}
+  >
+    Use the JSON parse function, `jparse()`, to parse a string value and produce a Map / List of values (or nested structures) which can be handled like any other first-class value type in NRQL.
+
+    `jparse()` takes two arguments:
+    * attribute - A JSON string value. 
+    * path - An optional string value that is used to directly reference a particular piece of the JSON within the `attribute` parameter. See the [JSON Parse Path Syntax Reference](#json-parse-path-syntax-reference) section below.
+
+
+    The `jparse()` function follows the [RFC 8259 format](https://datatracker.ietf.org/doc/html/rfc8259#section-2) to parse JSON values. When the `jparse()` function is used without the `path` parameter, it will return the deserialized JSON value.
+    
+
+    You can use the square-bracket syntax to pull out individual values from a `jparse()` result via a key/index, and map JSON keys directly to attributes using the `WITH` clause.
```

Comment:
"You can use square brackets" is less wordy and I believe conveys the same information.

Reply:
Sounds good, updated the text accordingly.